### DISCUSSION TITLE
Highlight and stiped not working after migration to v7

### DISCUSSION THREAD

**Discussion by petrvecera** [icflorescu/mantine-datatable#622] [Q&A] [answered]
I am trying to migrate the code to v7, I got most of the things working - but the highlight and striped doesn't work. It doesn't really make any sense why it should not work. 😕 

Any ideas, leads or anything? 

One of the tables
https://github.com/cohstats/coh3-stats/blob/migrate-mantine-v7/screens/leaderboards/leaderboards.tsx#L48C27-L49C11

Mantine table CSS imported https://github.com/cohstats/coh3-stats/blob/migrate-mantine-v7/pages/_app.tsx#L2

_Mantine rant. Btw I hate Mantine v7, the migration is extreme pain, almost the whole app doesn't work / doesn't look the same. You basically need to rewrite most of the stuff, I am like 10 hours in and have maybe 20% done? End of Mantine rant_ 

Btw thank you for your great library ❤️

**Comment by petrvecera** [comment, marked as answer]
You need to correctly import layouts https://icflorescu.github.io/mantine-datatable/styling/

**Comment by JustinGrote** [comment]
I was stuck here and you have to import the **layer** CSS, not the standard non-layer CSS as per the default [Mantine Styles](https://mantine.dev/styles/mantine-styles/) directions. It might help to add a callout on the datatable docs about this.
